base_mail_sender_bcc/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
```suggestion
# © 2017 Akretion (http://www.akretion.com)
# @author Alexis de Lattre <alexis.delattre@akretion.com>
# Copyright 2021 Quartile Limited
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Pactera had removed the copyright attributions. We should restore them.

Ref. https://github.com/cubicltd/akretion-odoo-usability/blob/16e550d0d5a3a34c74106c0f5e98cd610ff3d943/base_mail_sender_bcc/__manifest__.py#L2-L3